ui/src/stats.tsx

```diff
@@ -97,7 +97,7 @@ export const
         <div key={`${i}:${label}`} className={css.stat} style={statStyle}>
           {icon && (
             <div className={css.lhs} style={icon_color ? { color: cssVar(icon_color) } : undefined}>
-              <FontIcon className={css.icon} iconName={icon} />
+              <Icon className={css.icon} iconName={icon} styles={{ root: { '> span': { height: '100%'}}}}/>
```

Comment:
The font icon styling is not necessary anymore. Make sure the rendered icon is 45x45.
```suggestion
              <Icon iconName={icon} styles={{ root: { '> span': { height: '100%'}}}}/>
```

Reply:
Changed `font-size` -> `height` + `width`
The icon is not rendered if the `height` or `width` is not set for `<i />`